ext/standard/proc_open.c

```diff
@@ -1020,54 +1077,33 @@ PHP_FUNCTION(proc_open)
 			goto exit_fail;
 		}
 	}
-	newprocok = CreateProcessW(NULL, cmdw, &php_proc_open_security, &php_proc_open_security,
-		TRUE, dwCreateFlags, envpw, cwdw, &si, &pi);
+	newprocok = CreateProcessW(NULL, cmdw, &php_proc_open_security,
+		&php_proc_open_security, TRUE, dwCreateFlags, envpw, cwdw, &si, &pi);
 
 	if (suppress_errors) {
 		SetErrorMode(old_error_mode);
 	}
 
-	if (FALSE == newprocok) {
+	if (newprocok == FALSE) {
 		DWORD dw = GetLastError();
 		close_all_descriptors(descriptors, ndesc);
-		php_error_docref(NULL, E_WARNING, "CreateProcess failed, error code - %u", dw);
+		php_error_docref(NULL, E_WARNING, "CreateProcess failed, error code: %u", dw);
```

Comment:
this change should be revert I think

Reply:
Thanks. You feel the other format looks better?

Reply:
Not really, I'm mostly concerned about needless BC breaks :)
(those messages can be used e.g in userland test cases, better not break them)

Reply:
As far as PHP 8 is concerned, any and all error message changes are fine. They have changed so much that it makes no sense to be concerned about individual changes.

Reply:
All other messages in the file use a ` - ` also. And saving small annoying BC breaks is still nice.
As you want really of course!

Reply:
Personally I would rather change the other error messages in the same file to use a colon. (I think it looks better.) But whatever the maintainers want...

Reply:
I believe the colon is our standard format for this nowadays, but we can check with @kocsismate.

Reply:
Yep, I also believe the usage of `:` is much more widespread as a separator than `-`. And it also matches my personal preference more, so +1 for the change. :)